occurrence-integration-tests/pom.xml

```diff
@@ -167,26 +191,30 @@
 
     <!-- HBase Mini Cluster-->
     <dependency>
-      <groupId>org.apache.hbase</groupId>
-      <artifactId>hbase-testing-util</artifactId>
-      <version>${hbase.version}</version>
-    </dependency>
-    <dependency>
-      <groupId>org.apache.hadoop</groupId>
-      <artifactId>hadoop-minicluster</artifactId>
-      <version>${hadoop.version}</version>
+      <groupId>org.gbif.occurrence</groupId>
```

Comment:
shouldn't this be set in the parent pom as we do with the other occurrence dependencies?

Reply:
I meant not to have the 1.0-SNAPSHOT hardcoded here

Reply:
yes, good catch. I've done this now